tools/slang-reflection-test/slang-reflection-test-main.cpp

```diff
@@ -261,6 +273,74 @@ static void emitReflectionModifierInfoJSON(
     }
 }
 
+static void emitUserAttributeJSON(PrettyWriter& writer, slang::UserAttribute* userAttribute)
+{
+    write(writer, "{\n");
+    indent(writer);
+    write(writer, "\"name\": \"");
+    write(writer, userAttribute->getName());
+    write(writer, "\",\n");
+    for (unsigned int i = 0; i < userAttribute->getArgumentCount(); i++)
+    {
+        int intVal;
+        float floatVal;
+        char stringVal[1024];
+        size_t bufSize = sizeof(stringVal);
+        if (i > 0)
+            write(writer, ",\n");
+        write(writer, "\"arg"); write(writer, (SlangUInt)i); write(writer, "\": ");
+        if (userAttribute->getArgumentValueInt(i, &intVal) == 0)
```

Comment:
These should probably be using the `SLANG_SUCCEEDED(...)` macro instead of testing directly against a plain `0` constant, just to make clear that the functions are working with a `SlangResult`.

Reply:
Fixed.